FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/base/BaseComposeScreenTest.kt

```diff
@@ -68,4 +78,17 @@ abstract class BaseComposeScreenTest : BaseTest() {
         closeSoftKeyboard()
       )
   }
+
+  protected fun addAtt(att: File) {
```

Comment:
`protected fun addAttachment(file: File)` would be more convinent function naming i think.